src/Dfc.CourseDirectory.Functions/FixVenues/AnalysisCounts.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Dfc.CourseDirectory.Functions.FixVenues
+{
+    internal class AnalysisCounts
+    {
+        public int BatchSize { get; set; }
+        public int CorruptLocationsAnalysed { get; private set; }
+        public IList<FixCounts> FixCounts { get; private set; } = new List<FixCounts>();
+
+        public static AnalysisCounts GetCounts(IList<ApprenticeshipVenueCorrection> apprenticeshipVenueCorrections)
+        {
+            var locationCorrections = apprenticeshipVenueCorrections
+                .Where(r => r.ApprenticeshipLocationVenueCorrections.Any())
```

Comment:
Nit: I think this line is redundant

Reply:
you're right, and I've added a quick unit test to satisfy myself in a new commit, done